sites/mosul/configs/openmrs/initializer_config/ampathforms/F09-ITFC_Discharge_form.json

```diff
@@ -1057,10 +1067,9 @@
               "questionOptions": {
                 "rendering": "number",
                 "concept": "899a2fea-ba57-4744-9fcf-94f3e0d8d71f",
-                "min": 1,
+                "min": 0,
```

Comment:
Please confirm if the min is meant to be 1 from the metadata file.

Reply:
Yes @pirupius , you can verify it on the Row 63 of the ITFC Discharge form